contracts/seed/Seed.sol

```diff
@@ -625,7 +625,7 @@ contract Seed {
             therefore contributors can no longer withdraw their funding tokens.
         */
         require(
-            maximumReached || (minimumReached && block.timestamp >= endTime),
+            maximumReached || minimumReached,
```

Comment:
```suggestion
            minimumReached,
```
Suggestion to what they recommended, because
when `maximumReached` has not been reached (=`false`)
`minimumReached` is always `true`, so this evaluates to `true` always (in that case)

Reply:
Agree, the `maximumReached` is unnecessary here